api.go

```diff
@@ -113,7 +114,7 @@ func NewAPIFromConfig(configFile string) (*CacophonyAPI, error) {
 			return nil, err
 		}
 	}
-	api, err := NewAPI(conf.ServerURL, conf.Group, conf.DeviceName, password)
+	api, err := NewAPI(conf.ServerURL, conf.Group, conf.DeviceName, 0, password)
```

Comment:
Should this be passing the device id from the config file if it's available? Ultimately we want to just authenticate with the device id right? Then only registration needs to have the group and device name.

Reply:
ofcourse :) silly me